AdyenCard/Utilities/CardType/FallbackCardBrandProvider.swift

```diff
@@ -14,7 +14,7 @@ internal final class FallbackBinInfoProvider: AnyBinInfoProvider {
     internal func provide(for bin: String, supportedTypes: [CardType], completion: @escaping (BinLookupResponse) -> Void) {
         let result: [CardBrand] = supportedTypes.adyen.types(forCardNumber: bin).map { brand in
 
-            let cvcPolicy: CardBrand.CVCPolicy
+            let cvcPolicy: CardBrand.RequirementPolicy
             switch brand {
             case .laser,
```

Comment:
does it makes sense to move this into `CardType`? seems like this is directly connected to the card type. 
maybe we can also not use `default` in the `switch` if we decide to move this

Reply:
Could you eloborate on this one? Didn't get it

Reply:
I am keeping it as is since it's not related to this PR. But can be discussed for change on its own topic.